### ISSUE TITLE
About --ImageReader.single_camera_per_folder

### ISSUE THREAD

**Issue by koz7** [colmap/colmap#1075]
Hi, thank you always for your support.
I consider the following situation:
1) There is two cameras handled by two persons.
2) They move around independently.
3) So, the input path has the following structure:
[Input path]
|
|_camera1
|    |_image001
|    |_image002
|    |_...
|
|_camera2
     |_image001
     |_image002
     |_...

4) I need to specify intrinsic parameters for each camera, and optimize those two intrinsic parameters for the two cameras as well as poses simultaneously.

For that purpose, is the following command makes sense?
I looked your tutorial about "--ImageReader.single_camera_per_folder", but there seems difficult to understand how it specifies. 
I wonder, "--ImageReader.single_camera_per_folder 1" means "each camera folder has a single set of intrinsic parameters and can be optimized while reconstruction", doesn't it?

colmap feature_extractor \
--database_path [path]/database.db \
--image_path [input_path] \
--ImageReader.single_camera_per_folder 1 \

Thank you in advance.

**Comment by TroiiF** [comment]
Yes, you are right. The structure of images should be like this:
-| images_folder
--| camera1(sub folder1 of images_folder)
----| image1.jpg
----| image2.jpg
----| ...
--| camera2(sub folder2 of images_folder)
----| image1.jpg
----| image2.jpg
----| ...

**Comment by moshebou** [comment]
Hi,
A followup question - how can I specify the intrinsics parameters for each folder in the command line?

**Comment by MaxPalmer-UH** [comment]
In this case will it use the database with camera intrinsics specified there for each camera ID? If there is no database, how do you supply the intrinsics for each camera as @moshebou says? @TroiiF

**Comment by MaxPalmer-UH** [comment]
Also, how does it know which camera applies to each folder?
